scripts/collect_model_urls.py

```diff
@@ -0,0 +1,22 @@
+import pathlib
+import re
+import sys
+
+MODEL_URL_PATTERN = re.compile(r"https://download.pytorch.org/models/.*?[.]pth")
```

Comment:
> Code scanning warning: This regular expression has an unescaped '.' before 'pytorch.org', so it might match more hosts than expected.

This is indeed a correct warning. We should fix it.